src/lib/Libifl/ifl_impl.c

```diff
@@ -0,0 +1,788 @@
+ * @param[in] attrib -  pointer to attropl structure
+ * @param[in] extend - extend string to encode req
+ *
+ * @return	int
+ * @retval	0	success
+ * @retval	!0	error
+ *
+ */
+int
+pbs_manager(int c, int command, int objtype, char *objname,
+		struct attropl *attrib, char *extend) {
+	return (*pfn_pbs_manager)(c, command, objtype, objname,
+		attrib, extend);
+}
+
+/**
+ * @brief
+ *	Pass-through call to send move job request
+ *
+ * @param[in] c - connection handler
+ * @param[in] jobid - job identifier
+ * @param[in] destin - job moved to
+ * @param[in] extend - string to encode req
+ *
+ * @return      int
+ * @retval      0       success
+ * @retval      !0      error
+ *
+ */
+int
+pbs_movejob(int c, char *jobid, char *destin, char *extend) {
+	return (*pfn_pbs_movejob)(c, jobid, destin, extend);
+}
+
+/**
+ * @brief
+ *	-Pass-through call to send the MessageJob request and get the reply.
+ *
+ * @param[in] c - socket descriptor
+ * @param[in] jobid - job id
+ * @param[in] fileopt - which file
+ * @param[in] msg - msg to be encoded
+ * @param[in] extend - extend string for encoding req
+ *
+ * @return	int
+ * @retval	0	success
+ * @retval	!0	error
+ *
+ */
+int
+pbs_msgjob(int c, char *jobid, int fileopt, char *msg, char *extend) {
+	return (*pfn_pbs_msgjob)(c, jobid, fileopt, msg, extend);
+}
+
+/**
+ * @brief
+ *	-Pass-through call to send order job batch request
+ *
+ * @param[in] c - connection handler
+ * @param[in] job1 - job identifier
+ * @param[in] job2 - job identifier
+ * @param[in] extend - string to encode req
+ *
+ * @return      int
+ * @retval      0       success
+ * @retval      !0      error
+ *
+ */
+int
+pbs_orderjob(int c, char *job1, char *job2, char *extend) {
+	return (*pfn_pbs_orderjob)(c, job1, job2, extend);
+}
+
+/**
+ * @brief
+ *	-Pass-through call to send rerun batch request
+ *
+ * @param[in] c - connection handler
+ * @param[in] jobid - job identifier
+ * @param[in] extend - string to encode req
+ *
+ * @return      int
+ * @retval      0       success
+ * @retval      !0      error
+ *
+ */
+int
+pbs_rerunjob(int c, char *jobid, char *extend) {
+	return (*pfn_pbs_rerunjob)(c, jobid, extend);
+}
+
+/**
+ * @brief
+ *	-Pass-through call to release a hold on a job.
+ * 	really just an instance of the "manager" request.
+ *
+ * @param[in] c - connection handler
+ * @param[in] jobid - job identifier
+ * @param[in] holdtype - type of hold
+ * @param[in] extend - string to encode req
+ *
+ * @return      int
+ * @retval      0       success
+ * @retval      !0      error
+ *
+ */
+int
+pbs_rlsjob(int c, char *jobid, char *holdtype, char *extend) {
+	return (*pfn_pbs_rlsjob)(c, jobid, holdtype, extend);
+}
+
+/**
+ * @brief
+ *	-Pass-through call to send preempt jobs batch request
+ *
+ * @param[in] c - connection handler
+ * @param[in] preempt_jobs_list - list of jobs to be preempted
+ *
+ * @return      int
+ * @retval      0       success
+ * @retval      !0      error
+ *
+ */
+preempt_job_info*
```

Comment:
You should probably be consistent with other pointer return values and put a space between the type and pointer.